### PR TITLE
Add Dotty support

### LINKED ISSUE #126: Cross-publish for Dotty

### PR DESCRIPTION
**Pull request by rossabaker**
This depends on a milestone.  I propose we call it 1.1.0-M1.

Brings in sbt-spiewak.

Fixes #126.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,21 +1,25 @@
-organization := "org.http4s"
-name := "jawn-fs2"
+import de.heikoseeberger.sbtheader.{LicenseDetection, LicenseStyle}
 
-ThisBuild / crossScalaVersions := Seq("2.12.12", "2.13.3")
-ThisBuild / scalaVersion := crossScalaVersions.value.last
+ThisBuild / organization := "org.http4s"
```
Comment (rossabaker) [inline build.sbt:3]: 1.x is still published under org.http4s.  We'll need to change this on merge.

File: sonatype.sbt [hunk 2]
```diff
@@ -17,13 +17,14 @@ publishMavenStyle := true
...
 licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 
-homepage := Some(url("https://github.com/http4s/jawn-fs2"))
+homepage := Some(url("https://github.com/typelevel/jawn-fs2"))
 scmInfo := Some(
   ScmInfo(
-    url("https://github.com/http4s/jawn-fs2"),
-    "scm:git@github.com:http4s/jawn-fs2.git"
+    url("https://github.com/typelevel/jawn-fs2"),
+    "scm:git@github.com:typelevel/jawn-fs2.git"
```
Comment (rossabaker) [inline sonatype.sbt:24]: Even though this branch is still published to org.http4s, the metadata is out of date.

File: build.sbt [hunk 3]
```diff
@@ -25,4 +29,14 @@ publishTo := Some(
...
+headerLicense := {
+  val current = java.time.Year.now().getValue
+  val copyrightYear = startYear.value.fold(current.toString)(start => s"$start-$current")
+  LicenseDetection(
+    licenses.value.toList,
+    organizationName.value,
+    Some(copyrightYear),
+    headerLicenseStyle.value
+  )
+}
```
Comment (rossabaker) [inline build.sbt:42]: I can be talked out of these to go with the flow.  Without it, the copyright is just 2014, and it drones on for several lines.

**Comment by rossabaker** [comment]
Needs to opt out of MiMa against 1.0 on Dotty, but I'm going to bed.

**Comment by larsrh** [comment]
Should we merge this to `main` too?

**Comment by rossabaker** [comment]
Yes.  It's a pain, because the package is renamed on that branch.  I'll give it a shot.

**Pull request merged**